src/main.rs

```diff
@@ -7,18 +7,24 @@
 //! Check the f****(as in friendly) code.
 
 #[cfg(feature = "tui")]
-use anyhow::Result;
+use thiserror::Error;
+#[derive(Error, Debug)]
+pub enum MainError {
+    #[error("Failed to run everything 😔")]
+    InitingError,
```

Comment:
Initializing you mean?
What is `MainError`? This is the primary error? Does this means that all other error are secondary? Bad naming.

nACK

Reply:
oh boy